roop/state.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from roop import utilities
+
+PROCESSED_PREFIX = '_'
+processed_frames_cnt = 0
```

Comment:
Not a fan of this global counter... it should be calculated via a method everytime

Reply:
Well, I would prefer to make this counter as a class attribute (probably accessed through a getter). But since classes are not used here, I used a global variable. Anyway, it doesn't matter.